src/engraving/tests/compat114_data/fingering-ref.mscx

```diff
@@ -101,7 +101,7 @@
             <durationType>quarter</durationType>
             <Note>
               <Fingering>
-                <text>1</text>
+                <text><fontsize="8"/>1</text>
```

Comment:
Given the implementation of `extractFont`, I would expect to see `<font size="8"/>` here, i.e. with space. And I think I've never seen MuseScore write `fontsize` without space.

Reply:
fixed